### DISCUSSION TITLE
写在关闭讨论区后的几天

### DISCUSSION THREAD

**Discussion by phaedo** [WeNeedHome/SummaryOfLoanSuspension#937] [Ideas]
1. 对发起人说，关闭讨论区让此事热度直线下降。
2. 串联广泛分布的多方参与者十分困难，更谈不上统一行动。
3. 为了拯救下一个30年，建议开一个Telegram或discord群，发房产证截图进群，否则一盘散沙，毫无作为。
4. 各个银行已经发话，烂尾房断贷对他们毫无影响。各地政府也表示可以支持断贷缓冲。那么可以预见，如果热度下降后问题没有解决，那就失去了一次最好的机会。

**Comment by Accelerator6666** [comment]
银行那种打肿了充胖子的新闻发言毫无可信度，固然断贷的那一部分他们可能可以化解，但是因此带出来的连锁反应他们是肯定扛不住的。现在已经不仅仅是买房的人，还有那些和烂尾楼相关的合作单位也开始断贷了，这就是连锁反应。

**Comment by gyvao** [comment]
发起人没建，有个电报https://t.me/+duhfyQgraxAwYjFl

**Comment by davidliu1645** [comment]
发起人不是被办了吗

**Comment by TALK1949** [comment]
毫无影响……这也信？没影响封个什么劲……
